### DISCUSSION TITLE
Sample data for development

### DISCUSSION THREAD

**Discussion by farski** [tphakala/birdnet-go#139] [Q&A]
I'm hoping to start spending some time on UI work this week or next week. @tphakala I'm wondering what you do to populate development or seed data? The homepage obviously is dependent on have data from today, so if I grab a snapshot of my real DB and clips today, by tomorrow the homepage would be empty again making it hard to develop.

Do you have something you've been using to populate sample data for development?

If not, I could imagine it being helpful to have a few different seed data sets to mock up certain scenarios. For example, there are UI features I'm imagining that highlight less commonly scene species, so one data set that only includes similar species every day, and another where today always includes a rare bird. I'm sure there are other helpful edge cases to be able to generate on demand as well.

I can work on this before diving into the UI stuff, just wanted to see if you already had anything started.

**Comment by tphakala** [comment]
I just copy fresh db and latest clips over from my prod instance to dev, on a daily basis really, with ssh key auth this takes just a few seconds

copy database from 192.168.4.159 to localhost

`scp user@192.168.4.159:birdnet.db .
`

copy this months audio clips to localhost

`rsync -ru --progress "user@192.168.4.159:clips/2024/05" clips/2024/05/
`
